bochs/bx_debug/dbg_main.cc

```diff
@@ -2283,6 +2283,46 @@ void bx_dbg_lyt(void)
   bx_nest_infile(layoutpath);
 }
 
+void bx_dbg_set_magic_bp_mask(Bit8u mask)
+{
+  bx_dbg.magic_break = mask;
+  dbg_printf("new magic breakpoint mask created: 0x%x\n", mask);
```

Comment:
this debug printf is useless
I would parse the mask instead and print smth like: magic break will break on registers "bp, si"
it also gives you some indication if everything was parsed correctly